### PR TITLE
Fix relationship resume and proccessing test case

### PR DESCRIPTION
**Pull request by misha3674**
i divide the change into two branches fix_relationship_resume and add_testcase, but all changes add in this pull request. Branch 'add_testcase' is not complete. It is done as soon as possible.

### CODE REVIEW

**Comment by nazarello** [comment]
Міша привіт, це Назар, я недавно до вас приєднався на проект, якщо тобі не
важко скинь, будь ласка, розклад пар по проекту.

2017-11-03 20:30 GMT+02:00 Maydanskiy Misha <notifications@github.com>:

**Comment by misha3674** [comment]
Wednesday 18,00-21,00 MagentaRoom;  Saturday;

File: app/Models/Currency.php [hunk 1]
```diff
@@ -15,4 +15,9 @@ public function getCurrencies()
 
         return $currencies;
     }
-}
\ No newline at end of file
+
+    public function Resumes()
+    {
+        return $this->hasMany('App\Models\Resumes', 'currency_id')->getResults();
```
Comment (Quicks) [inline app/Models/Currency.php]: scope should return query, not results

File: app/Models/Industry.php [hunk 2]
```diff
@@ -18,8 +18,10 @@ public function GetVacancies()
         return $this->hasMany('App\Models\Vacancy','branch')->latest('updated_at');
     }
 
-    ////////////////////////////////////////////////////////////////////////////////////////////////////
-    //Scopes
+    public function Resumes()
+    {
+        return $this->hasMany('App\Models\Resumes', 'industry')->getResults();
```
Comment (Quicks) [inline app/Models/Industry.php]: scope should return query, not results

File: app/Models/Resume.php [hunk 3]
```diff
@@ -24,16 +24,24 @@ public function getResumes()
...
     {
-        $user = $this->belongsTo('App\Models\User','id_u')->first();
-        return $user;
+        return $this->belongsTo('App\Models\User', 'id_u')->getResults();
     }
-    public function ReadUser()
+
+    public function City()
+    {
+        return $this->belongsTo('App\Models\City', 'city')->getResults();
```
Comment (Quicks) [inline app/Models/Resume.php]: scope should return query, not results

**Pull request closed without merging**